ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/config/Cli.scala

```diff
@@ -67,7 +67,7 @@ object Cli {
       .optional()
       .action { case (_, config) => config.copy(latencyTest = true) }
 
-    opt[Long]("max-latency")
+    opt[Long]("max-latency-millis")
```

Comment:
How about
```suggestion
    opt[Duration]("max-latency")
```
and then doing
--max-latency 5ms